src/theory/quantifiers/inst_strategy_enumerative.cpp

```diff
@@ -70,225 +70,244 @@ void InstStrategyEnum::check(Theory::Effort e, QEffort quant_e)
     doCheck = quant_e == QEFFORT_LAST_CALL;
     fullEffort = !d_quantEngine->hasAddedLemma();
   }
-  if (doCheck)
+  if (!doCheck)
   {
-    double clSet = 0;
-    if (Trace.isOn("fs-engine"))
-    {
-      clSet = double(clock()) / double(CLOCKS_PER_SEC);
-      Trace("fs-engine") << "---Full Saturation Round, effort = " << e << "---"
-                         << std::endl;
-    }
-    int addedLemmas = 0;
-    for (unsigned i = 0;
-         i < d_quantEngine->getModel()->getNumAssertedQuantifiers();
-         i++)
+    return;
+  }
+  double clSet = 0;
+  if (Trace.isOn("fs-engine"))
+  {
+    clSet = double(clock()) / double(CLOCKS_PER_SEC);
```

Comment:
Don't we want to be using `TimerStat` for this kind of stuff?

Reply:
I dont know, but this is outside of the scope of this PR (this is my convention in quantifiers).